### DISCUSSION TITLE
Step-by-step practice problem mode (Solve problem with you / Problem walkthrough)

### DISCUSSION THREAD

**Discussion by zachberry** [ucfopen/Obojobo#1845] [Ideas]
I think we'd still like to have this feature, so creating a discussion to keep track of it. See this stale issue for more information: https://github.com/ucfopen/Obojobo/issues/563

**Comment by paradisojr** [comment]
Having a multi-part question can kind of be simulated in Practice already, but having a multi-part ‘scored’ Assessment question cannot be simulated well because there is no overarching text area above the questions where a prompt can be placed. The most intricate version, of course, would have the numeric calculations churning in the background as the problem is progressively being solved. This has definitely been requested by STEM faculty I work with in Obo.
